go.sum

```diff
@@ -154,6 +154,8 @@ github.com/ethereum/c-kzg-4844 v1.0.0 h1:0X1LBXxaEtYD9xsyj9B9ctQEZIpnvVDeoBx8aHE
 github.com/ethereum/c-kzg-4844 v1.0.0/go.mod h1:VewdlzQmpT5QSrVhbBuGoCdFJkpaJlO1aQputP83wc0=
 github.com/ethereum/go-ethereum v1.14.8 h1:NgOWvXS+lauK+zFukEvi85UmmsS/OkV0N23UZ1VTIig=
 github.com/ethereum/go-ethereum v1.14.8/go.mod h1:TJhyuDq0JDppAkFXgqjwpdlQApywnu/m10kFPxh8vvs=
+github.com/ethereum/go-ethereum v1.14.13 h1:L81Wmv0OUP6cf4CW6wtXsr23RUrDhKs2+Y9Qto+OgHU=
```

Comment:
The previous `github.com/ethereum/go-ethereum v1.14.8` entry has not been deleted. Running `go mod tidy` locally confirmed that it has become redundant. 

While it's going to work, it would be great to clean up old versions because it seems version bumps will happen quite often.

@zktaiga can you please check if there's some renovate option to clean up after version bump?

Reply:
seems resolved